setup.py

```diff
@@ -33,6 +33,8 @@
         'urllib3 >= 1.25.3',
         'python-dateutil',
         'requests',
+        'click',
+        'google-cloud-storage',
```

Comment:
```suggestion
```

These requirements are for the install-able (through openapi) python module, not for the scripts. Traditionally scripts are run through the analysis-runner which already has these modules installed.